Find a functionally equivalent code:android.os.StrictMode.writeGatheredViolationsToParcel:COMMENT
Method Modifier: default     static      
Comment:/* package */

@@ -5,11 +5,16 @@
     } else {
         p.writeInt(violations.size());
         for (int i = 0; i < violations.size(); ++i) {
+            int start = p.dataPosition();
             violations.get(i).writeToParcel(p, 0);
+            int size = p.dataPosition() - start;
+            if (size > 10 * 1024) {
+                Slog.d(TAG, "Wrote violation #" + i + " of " + violations.size() + ": " + (p.dataPosition() - start) + " bytes");
+            }
         }
         if (LOG_V)
             Log.d(TAG, "wrote violations to response parcel; num=" + violations.size());

